app/views/shared/_problem_order_details.html.haml

```diff
@@ -21,8 +23,8 @@
         %td.centered= link_to order_detail.id,
           manage_order_detail_path(order_detail),
           class: "manage-order-detail"
-
-        %td= human_datetime(order_detail.order.ordered_at)
+        %td= problem_order_date(order_detail, local_assigns[:reservation_list_page])
```

Comment:
This was needed because there are two pages that share this same partial (orders and reservations) and we want to display order date for a problem orders, but we want to see actual start time for a problem reservation.